pkg/proto/buildqueuestate/buildqueuestate.proto

```diff
@@ -235,12 +235,21 @@ message PlatformQueueState {
 }
 
 message InvocationState {
+  message InvocationObjectCount {
+    // The number of objects that are part of this invocation.
+    uint32 direct = 1;
+
+    // The number of objects that are part of this invocation and all its descendants.
```

Comment:
I think it would make more sense for this field to only include the count for the descendants, not the invocation itself. The advantage of doing that is that it sort of ["makes impossible states impossible"](https://kentcdodds.com/blog/make-impossible-states-impossible). Namely, if we include the value of the invocation itself, we'd also have to describe/enforce/guarantee that `direct <= indirect`.